app/views/cart/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @items.each do |item| %>
+
+<img src="<%= item.image_path %>" alt="Item" style="width:100px;height:80px;"> <br>
+
+  Title: <%= item.title %> <br>
+  Description: <%= item.description %> <br>
+  Price: <%= item.price %> <br>
```

Comment:
Should we put this text inside a tag? I feel a little weird about the text hanging around without a tag